### PR TITLE
Load Balancing Algorithm Options Added

### PR DESCRIPTION
**Pull request by kimikowang**

### CODE REVIEW

File: pkg/api/rule.go [hunk 1]
```diff
@@ -59,6 +59,7 @@ type Backend struct {
 	Weight  float64  `json:"weight,omitempty"`
 	Timeout float64  `json:"timeout,omitempty"`
 	Retries int      `json:"retries,omitempty"`
+	Lb_type string 	 `json:"lb_type,omitempty"`
```
Comment (ijsnellf) [inline pkg/api/rule.go]: Variable naming style:
```
LbType string 	 `json:"lb_type,omitempty"`
```

**Review by rshriram — CHANGES_REQUESTED** [review]
You still have to pass this to envoy config right?
Can you verify that the generated config file for envoy actually has the right load balancing algorithm?

**Review by rshriram — CHANGES_REQUESTED** [review]
Please remove productpage binary and the cli/i18n/resources.go. Otherwise it looks good.

**Comment by kimikowang** [comment]
Changes have been made to sidecar/proxy/envoy/manager.go, where LbType value in backend rule will be included in cluster building to generate the config file.

**Comment by rshriram** [comment]
See my comment above. You have included irrelevant files in your commit.

**Review by rshriram — CHANGES_REQUESTED** [review]
And you added another binary.

**Review by rshriram — COMMENTED** [review]
i18n_resources.go is still there.

**Review by rshriram — APPROVED** [review]

**Pull request merged**